src/qz/printer/action/WebApp.java

```diff
@@ -145,6 +150,24 @@ public WebApp() {
 
     /** Starts JavaFX thread if not already running */
     public static synchronized void initialize() throws IOException {
+        if(Constants.JAVA_VERSION.greaterThanOrEqualTo(Version.valueOf("11.0.0"))) {
+            // Monocle for Windows/MacOS relies on hw pipeline
+            if(!SystemUtilities.isWindows() && !SystemUtilities.isMac()) {
+                // Monocle for Linux relies on sw pipeline (tailored for embedded)
+                System.setProperty("prism.order", "sw");
+            }
+
+            System.setProperty("javafx.platform", "monocle"); // Standard JDKs
+            System.setProperty("glass.platform", "Monocle"); // Headless JDKs
+            System.setProperty("monocle.platform", "Headless");
+            System.setProperty("headless.geometry", "15360x15360-32"); //TODO - determine based on memory, >=1g for 15 (~10 at 500m)
```

Comment:
I had to reduce the size to get the image capture to succeed on my Windows 10 and Ubuntu 18.0.4 (each have 4GB of RAM)

```diff
- System.setProperty("headless.geometry", "15360x15360-32");
+ System.setProperty("headless.geometry", "8000x8000-32");
```

Reply:
Headless geometry should be dynamic based on memory available to qz now. From what I saw with setting `-XmxNN` monocle needs about 100mb per 1200 pixels, so that's what the calculation is based of off for now.

Reply:
Marking resolved per 17e8d4b.